resources/application-connector/charts/connectivity-certs-controller/README.md

```diff
@@ -7,8 +7,8 @@ The Connectivity Certs Controller fetches the client certificate and the root CA
 You can set the following parameters in the Connectivity Certs Controller chart:
 - **appName** - Name of the Controller used during registration. The default value is `connectivity-certs-controller`.
 - **namespace** - Namespace in which the Controller creates the Secrets that store certificates. The default Namespace is `kyma-integration`.
-- **clusterCertificatesSecret** - Name of the Secret which stores the client certificate and key. The default name is `cluster-client-certificates`.
-- **caCertificatesSecret** - Name of the Secret which stores the CA certificate. The default name is `nginx-auth-ca`.
+- **clusterCertificatesSecret** - Namespace and Name of the Secret which stores the client certificate and key in the format `Namespace/Name`. The default name is `kyma-integration/cluster-client-certificates`.
+- **caCertificatesSecret** - Namespace and Name of the Secret which stores the CA certificate in the format `Namespace/Name`. The default name is `istio-system/application-connector-ca-certs`.
```

Comment:
```suggestion
- **caCertificatesSecret** - Namespace and the name of the Secret which stores the client certificate and key. Requires the `Namespace/secret name` format. The default value is `istio-system/application-connector-ca-certs`.
```

Reply:
Changed it to `- **caCertificatesSecret** - Namespace and the name of the Secret which stores the CA certificate. Requires the `Namespace/secret name` format. The default value is `istio-system/application-connector-ca-certs`.` as it does store CA certificate, not the client one.